test/e2e/lib/install.bash

```diff
@@ -22,12 +22,8 @@ function uninstall_tiller() {
 }
 
 function install_flux_with_helm() {
-  local create_crds='true'
-  if kubectl get crd fluxhelmreleases.helm.integrations.flux.weave.works helmreleases.flux.weave.works > /dev/null 2>&1; then
-    # CRDs existed, don't try to create them
-    echo 'CRDs existed, setting helmOperator.createCRD=false'
-    create_crds='false'
-  fi
+
+  kubectl apply -f https://raw.githubusercontent.com/fluxcd/helm-operator/master/deploy/flux-helm-release-crd.yaml
```

Comment:
I would point this to a tagged release as otherwise the tests can become flakey due to upstream changes.

Given that within Flux the CRD is only used to automate things, I think the maintenance is low (or can be automated if proven tiresome by e.g. fetching the latest published semver version).

Reply:
Done